copter/source/docs/radio-failsafe.rst

```diff
@@ -4,149 +4,95 @@
 
-Set the "FS Pwm" value to be:
-
--  at least 10 PWM higher than your Channel 3's PWM value when the
-   throttle stick is fully down and the transmitter is **off**
--  at least 10 lower than your channel 3's PWM value when the throttle
-   stick is fully down and the transmitter is **on**\ 
+..  youtube:: qf8YinLKQww
+    :width: 100%
 
--  *above 910*
 
-.. image:: ../images/RadioFailsafe_MPSetup.png
-    :target: ../_images/RadioFailsafe_MPSetup.png
+Many receivers allow for the failsafe positions to be set either by simply pressing a button on the receiver, or directly from the transmitter. In this case, the transmitter is temporarily adjusted to output a throttle signal below the normal low idle position (if low idle is 1000us, then 990uS would be sent) setting this as the failsafe value for the receiver, then adjusting low stick back to normal idle position. This below throttle idle value is then set for :ref:`FS_THR_VALUE<FS_THR_VALUE>`, as discussed below.
 
-Set-up for No-Signal method
----------------------------
+Receiver configuration for No-Signal method
+-------------------------------------------
+**"No Signal" method** - the receiver stops sending signals to the flight controller.  This is the preferred method and is how most modern FrSky receivers operate. Below is the setup method for a FlySky 9 channel transmitter with FrSky D4R-II receiver which uses the "No Signal" method.
 
 ..  youtube:: FhKREgqjCpM
     :width: 100%
 
-Above is the setup method for a FlySky 9 channel transmitter with FrSky
-D4R-II receiver which uses the "No Signal" method.
 
-Similar to the "Low-Throttle" method, with the LiPo battery
-disconnected:
+Parameter Configuration
+========================
+
+The :ref:`FS_THR_ENABLE<FS_THR_ENABLE>` parameter can also be set using the Mission Planner *failsafe options* dropdown in the Initial Setup >> Mandatory Hardware >> Failsafe menu.
 
--  Connect your flight controller to the mission planner and select
-   Initial Setup >> Mandatory Hardware >> Failsafe.
--  Set the Failsafe Options to one of the three options:
+-  **Disabled** (Value 0) will disable the radio failsafe entirely.
+-  **Enabled Always RTL** (Value 1) will switch the copter to RTL Mode.  If the GPS position is not usable, the copter will change to Land Mode instead.
+-  **Enabled Continue with Mission in Auto Mode (Deprecated in 4.0+)** (Value 2) will ignore the failsafe in an Auto Mode mission. Otherwise, it will behave the same as *Enabled Always RTL*. This option no longer exists in ArduCopter 4.0. Instead, see the :ref:`FS_OPTIONS<FS_OPTIONS>` parameter for this function. Setting this value in Copter4.0 and later version will automatically be converted and set to (Value 1) and the :ref:`FS_OPTIONS<FS_OPTIONS>` will be modified to include bit () in the bitmask for "".
```

Comment:
one addition needed...."will be modified to include bit (0) in the bitmask for "Continue if in AUTO Mode on Radio Failsafe".

Reply:
Good catch. I see a few spelling and grammar errorz in there too.  I'll do that this evening.